CMakeCPack.cmake

```diff
@@ -60,7 +60,7 @@ set(CPACK_PACKAGE_ICON "${CMAKE_SOURCE_DIR}/resources\\\\FreeRDP_Install.bmp")
 set(CPACK_NSIS_MUI_ICON "${CMAKE_SOURCE_DIR}/resources\\\\FreeRDP_Icon_96px.ico")
 set(CPACK_NSIS_MUI_UNICON "${CMAKE_SOURCE_DIR}/resource\\\\FreeRDP_Icon_96px.ico")
 
-set(CPACK_COMPONENTS_ALL client server libraries headers)
+set(CPACK_COMPONENTS_ALL client server libraries headers symbols tools)
```

Comment:
Since the symbols are only defined on windows with MSVC we should maybe use the same if here as well.
I'm not sure if this can't cause problems otherwise.